tests/cts_tests/interpreter/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+add_custom_target(interpreter_cts_tests)
+add_dependencies(tests interpreter_cts_tests)
+
+function(shrimp_cts_test test_name)
+	set(TEST_BUILD_DIR ${CMAKE_CURRENT_BINARY_DIR}/${test_name})
+	set(TEST_SOURCE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/${test_name}.shr)
+	file(MAKE_DIRECTORY ${TEST_BUILD_DIR})
+
+	add_custom_command(COMMAND cd ${TEST_BUILD_DIR} && ${PROJECT_BINARY_DIR}/assembler/assembler < ${TEST_SOURCE_PATH}
+		OUTPUT ${TEST_BUILD_DIR}/${test_name}.imp
```

Comment:
`OUTPUT ${TEST_BUILD_DIR}/out.a` ?

Reply:
We decided that our binary file extension is `imp`